app/processing/invoice-line/get-invoice-line.js

```diff
@@ -1,34 +1,10 @@
-const schema = require('./invoice-line-schema')
-
 const getPositiveInvoiceLineByFundingCodeAndPaymentRequestId = require('./get-positive-invoice-line-by-funding-code-and-payment-request-id')
-const getReductions = require('./get-reductions')
-
-const QUARTER = 0.25
-const MIN_PAYMENT_VALUE = 0
+const validateInvoiceLine = require('./validate-invoice-line')
+const mapInvoiceLine = require('./map-invoice-line')
```

Comment:
The retrieval and validation of negative invoice lines is hidden away in here.  Not saying that's wrong but just raising it as a consideration as to whether it's the intention.

Reply:
It is intentional, the retrieval in this context is limited to `mapInvoiceLine` block, since this block has been abstracted from the parent function `getInvoiceLine`, its direct and wholly owned dependences in the parent function are also abstracted as thus: 
https://github.com/DEFRA/ffc-pay-statement-constructor/blob/122240-standardise-object-retrieval-in-statement-constructor/app/processing/invoice-line/map-invoice-line.js